examples/working-directly-with-json/working_directly_with_json.md

```diff
@@ -1,11 +0,0 @@
-# Working directly with JSON
```

Comment:
Why deleting this file in this PR?

Reply:
This is an already added file, right? That's why it's showing deletion as a diff. Can you undo this change?

Reply:
Done.